packages/proto/package.json

```
@@ -0,0 +1,86 @@
+{
+  "name": "@waku/proto",
+  "version": "0.0.1",
+  "description": "Protobuf definitions for Waku",
+  "types": "./dist/index.d.ts",
+  "module": "./dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    }
+  },
+  "type": "module",
+  "author": "Waku Team",
+  "homepage": "https://github.com/waku-org/js-waku/tree/master/packages/proto#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/waku-org/js-waku.git"
+  },
+  "bugs": {
+    "url": "https://github.com/waku-org/js-waku/issues"
+  },
+  "license": "MIT OR Apache-2.0",
+  "keywords": [
+    "waku",
+    "decentralized",
+    "secure",
+    "communication",
+    "web3",
+    "ethereum",
+    "dapps",
+    "privacy"
+  ],
+  "scripts": {
+    "build": "run-s build:**",
+    "build:esm": "tsc",
+    "build:bundle": "rollup --config rollup.config.js",
+    "proto": "rimraf src/*.ts; protons src/*.proto",
+    "check:tsc": "tsc -p tsconfig.dev.json",
+    "prepublish": "npm run build",
+    "reset-hard": "git clean -dfx -e .idea && git reset --hard && npm i && npm run build",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier . --write",
+    "fix:lint": "eslint src/*.ts --fix"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "dependencies": {
+    "protons-runtime": "^3.1.0"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^22.0.0",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-node-resolve": "^13.3.0",
+    "@typescript-eslint/eslint-plugin": "^5.8.1",
+    "@typescript-eslint/parser": "^5.8.1",
+    "cspell": "^5.14.0",
+    "eslint": "^8.6.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-functional": "^4.0.2",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-prettier": "^4.0.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.1.1",
+    "rollup": "^2.75.0",
+    "typescript": "^4.6.3",
+    "protons": "^5.1.0"
+  },
+  "typedoc": {
+    "entryPoint": "./src/index.ts"
+  },
+  "files": [
+    "dist",
+    "bundle",
+    "src/*.ts",
+    "src/lib/**/*.ts",
+    "src/proto/**/*.ts",
```

```suggestion
    "src/**/*.ts",
```

To ensure files in subfolder (if we introduce them) are picked up